troposphere/sagemaker.py

```diff
@@ -87,6 +87,9 @@ class NotebookInstance(AWSObject):
     resource_type = "AWS::SageMaker::NotebookInstance"
 
     props = {
+        'AcceleratorTypes': ([basestring], False),
+        'AdditionalCodeRepositories': ([basestring], False),
+        'DefaultCodeRepository': ([basestring], False),
```

Comment:
DefaultCodeRepository should be changed to just 'basestring' instead of '[basestring]'.

Reply:
Thanks for the review. Have changed it.